app/views/govuk_publishing_components/components/_accordion.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
+  id ||= "default-id-#{SecureRandom.hex(4)}"
+  additional_classes ||= []
+  wrapper_classes = additional_classes.insert(0, 'govuk-accordion gem-c-accordion').join(' ')
+%>
+<% if defined?(items) %>
+  <% if items.present? %>
+    <div class="<%= wrapper_classes %>" data-module="accordion" id="<%= id %>">
+      <% items.each_with_index do |item, i| %>
+        <%
+          index = i + 1
+          item[:data_attributes] ||= nil
+        %>
+        <div class="govuk-accordion__section">
+          <div class="govuk-accordion__section-header">
+            <h2 class="govuk-accordion__section-heading">
```

Comment:
Might be worth considering a parameter to change the heading level here.

Reply:
Making the heading level configurable was discussed in the  [GOV.UK Frontend accordion pull request](https://github.com/alphagov/govuk-frontend/pull/958#issuecomment-432990753), but hasn't been implemented. 

I'll take a look and see if any work I do could be fed back into the accordion in Frontend - but I'd prefer it to be part of a separate piece of work so it doesn't block this pull request.

Reply:
https://github.com/alphagov/govuk_publishing_components/blob/master/lib/govuk_publishing_components/presenters/heading_helper.rb

Reply:
Thanks! I've added this in, so a `heading_level` parameter can now be used.